### PR TITLE
What is a state machine updates

### PR DESCRIPTION
**Pull request by laurakalbag**
This PR tweaks text, adds images, embedded machines, links state machine concepts to further docs, and adds further info about Stately.

### CODE REVIEW

File: blog/2023-10-02-what-is-a-state-machine/index.mdx [hunk 1]
```diff
@@ -1,59 +1,102 @@
...
-slug: what-is-a-state-machine
-# image:
-date: 2023-10-02
+slug: 2023-10-05-what-is-a-state-machine
+image: /blog/2023-10-05-what-is-a-state-machine.png
+date: 2023-10-05
 ---
 
-You might have heard of the term “state machine” or “finite-state machine” before. If you look it up [on Wikipedia](https://en.wikipedia.org/wiki/Finite-state_machine), you’ll read that a state machine is described as a “mathematical model of computation”, followed by a bunch of symbols and computer science jargon that doesn’t immediately seem to apply your team’s work.
+You might have heard of “state machine” or “finite-state machine” before. [Look up _state machine_ on Wikipedia](https://en.wikipedia.org/wiki/Finite-state_machine). You’ll read that a state machine is described as a “mathematical model of computation,” followed by a bunch of symbols and computer science jargon that doesn’t immediately seem to apply to your team’s work.
```
Comment (davidkpiano) [inline blog/2023-10-02-what-is-a-state-machine/index.mdx:11]:
```suggestion
You might have heard of “state machine” or “finite-state machine” before. [If you look up _state machine_ on Wikipedia](https://en.wikipedia.org/wiki/Finite-state_machine), you’ll read that a state machine is described as a “mathematical model of computation,” followed by a bunch of symbols and computer science jargon that doesn’t immediately seem to apply to your team’s work.
```

File: blog/2023-10-02-what-is-a-state-machine/index.mdx [hunk 2]
```diff
@@ -80,41 +123,97 @@ But there are also other events that can happen:
...
+State machines clearly define all possible states, events, and transitions, allowing testers to simulate specific events and verify if transitions to specific states occur as expected. Since state machines are directed graphs, they act like a “map” for your app logic. This map enables the automatic generation of many different “paths” through the state machine. Each path represents a potential use case, even those that may initially go unnoticed, such as edge cases. Exhaustive test path generation is also possible so that you can test every possible path through the state machine.
+
+_Our [export to Stories feature](/docs/export-as-code#export-formats) enables you to generate the happy paths (stories) for your app._
+
+```md
```
Comment (davidkpiano) [inline blog/2023-10-02-what-is-a-state-machine/index.mdx:168]: Can we show this as actual markdown (e.g. in a blockquote)?

Comment (laurakalbag) [reply blog/2023-10-02-what-is-a-state-machine/index.mdx]: Yes! Good point 😆
